### PR TITLE
Facade First Commit

### PR DESCRIPTION
**Pull request by duygu2**
Fixes #13

### CODE REVIEW

File: src/main/main.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (huseyinbabal) [inline src/main/main.iml:1]: This files should go to .gitignore file

File: src/test/test.iml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (huseyinbabal) [inline src/test/test.iml:1]: This file should go to .gitignore file

**Comment by huseyinbabal** [comment]
@duygu2 you are in the correct path that you added iml to gitignore, but could in order to apply new gitignore to your project, please do the followings on your local;

- `git rm -r --cached .`
- `git add .`
- `git commit -m "gitignore apply"`

**Comment by duygu2** [comment]
@huseyinbabal thanks, i made the changes you said.

**Review by huseyinbabal — APPROVED** [review]
@duygu2 thanks for the prompt reply 🥇

**Pull request merged**